linux/jdk/redhat/src/main/packaging/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM fedora:38
+ARG CONTAINER_REGISTRY=""
+
+FROM ${CONTAINER_REGISTRY}tianon/gosu as gosu
```

Comment:
historical Q but why from tianon (he's the DockerHub maintainer sure, but it seems odd we're referencing his registry and not an org based one.

Reply:
`gosu` is a tool he created to deal with `sudo`, `su` issues when executing different processes. I think there are different alternatives instead of using `gosu` but I think @gdams might have a better explanation for the question.

Reply:
@gdams - can you chime in on this one?

Reply:
that's correct, gosu is a tool that tianon maintains